undertow/api/src/main/java/org/wildfly/swarm/undertow/UndertowFraction.java

```diff
@@ -134,6 +172,9 @@ public void initialize(Fraction.InitContext initContext) {
         initContext.socketBinding(
                 new SocketBinding("https")
                         .port(SwarmProperties.propertyVar(UndertowProperties.HTTPS_PORT, "8443")));
+        initContext.socketBinding(
+                new SocketBinding("ajp")
+                        .port(SwarmProperties.propertyVar(UndertowProperties.AJP_PORT, "8009")));
```

Comment:
@kenfinnigan Thank you for checking. I agree with your comment of the former. For users who don't use AJP, it may be a little bit overhead. I'll re-sent the p-r.